main.go

```diff
@@ -104,15 +115,71 @@ func initLogging() (*os.File, error) {
 	return file, nil
 }
 
+// extension specific PublicSettings
+type PublicSettings struct {
+	Script   string   `json:"script"`
+	FileURLs []string `json:"fileUris"`
```

Comment:
FileUris are not needed for our extension, they are specific to the Custom Script Extension. You can keep them for now but we'll have to remove them eventually.